oeps/oep-0026-arch-xapi-integration.rst

```diff
@@ -0,0 +1,483 @@
+  introduction of the `Anonymized User ID`_.
+
+  Enterprises and other organizations may want to use this xAPI integration framework
+  to access data for their users. For those use cases, sharing PII may be required.
+  We have chosen to keep those use cases in mind, but not target them initially, with
+  the understanding that future work would be needed to address those needs.
+
+* **Deferring implementation of an LRS** - As mentioned above, we are consciously
+  postponing implementation of an Open edX specific LRS at this time. Although the
+  need for an LRS may be forthcoming, this initial iteration defers this work.
+
+  As a consequence, adaptive engines may need to maintain their own LRS if they need
+  to refer back to previous events. Given our business research to date, it seems
+  many adaptive engines are already maintaining their own custom-optimized storage
+  of event data.
+
+.. _IMS Global: https://www.imsglobal.org/
+.. _Caliper Sensor APIs: https://www.imsglobal.org/caliper-analytics-v1-public-repos-sensor-apis
+.. _LTI: http://www.imsglobal.org/activity/learning-tools-interoperability
+
+Open edX Events
+---------------
+
+Currently, the Open edX system supports and maintains events that are sent to tracking
+logs, as described in `Tracking Log Events`_.
+
+Prioritized List of Events
+==========================
+
+For this first iteration, we will focus primarily on the following events:
+
+- **Enrollment events**
+
+  + `edx.course.enrollment.activated <http://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#edx-course-enrollment-activated-and-edx-course-enrollment-deactivated>`_.
+       Whenever a learner enrolls in a course.
+  + `edx.course.enrollment.deactivated <http://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#edx-course-enrollment-activated-and-edx-course-enrollment-deactivated>`_.
+       Whenever a learner unenrolls from a course.
+
+- **Problem interaction events**
+
+  + `edx.grades.problem.submitted <http://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/course_team_event_types.html#edx-grades-problem-submitted>`_.
+      Whenever a learner submits any problem.
+  + `problem_check <http://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#problem-check>`_.
+       Whenever a learner's answer to a problem is checked.
+  + `showanswer <http://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#showanswer>`_.
+       Whenever a learner is shown the answer to a problem.
+  + `edx.problem.hint.demandhint_displayed <http://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/student_event_types.html#edx-problem-hint-demandhint-displayed>`_.
+       Whenever a learner requests a hint to a problem.
+
+- **Video events**
```

Comment:
There are a few video-related events missing from the list. The ones I can think of right now are changing the transcript language, changing the video speed, changing the volume, opening/closing the captions or the transcript, downloading the videos, and downloading the transcripts. We might not need all of those for this iteration, but I think getting the language could be important.